modules/lofreq/1.0/config/default.yaml

```diff
@@ -8,6 +8,8 @@ lcr-modules:
             sample_bai: "__UPDATE__"
             lofreq_filter: "{MODSDIR}/src/bash/lofreq_filter.sh"
 
+        tool: "lofreq" # included into the naming of output files, cand be updated to contain the module version
```

Comment:
Isn't the tool name (i.e. "lofreq") already in the config at a higher level? I could be wrong.

Reply:
It is, yes. I can simply do `CFG` here, but that would not be customizable. I think I created more confusion than the benefits by including this wildcard. I thought we were considering adding the tool name (and, in the future, possibly tool's version), so if the files get moved out of the directory, this information will follow with the file. My intention here was to create a placeholder, and if we will decide later to include the module version, it can be easily done here. I think at this point, it will be better if I explicitly say the tool name in the file naming, and we can modify this to be a wildcard at a later point, if needed.